frontend/pages/manage/users/[slug]/index.tsx

```diff
@@ -0,0 +1,107 @@
+import PageStats from '@/components/atoms/PageStats'
+import { useState } from 'react'
+import Avatar from 'react-avatar'
+
+interface UserType {
+    id: number
+    first_name: string
+    last_name: string
+    avatar: string
+    question_count: number
+    answer_count: number
+    vote_count: number
+}
+
+const user: UserType = {
+    id: 1,
+    first_name: 'John',
+    last_name: 'Doe',
+    avatar: 'John Doe',
+    question_count: 10,
+    answer_count: 5,
+    vote_count: 20,
+}
+
+const getActiveTabClass = (status: boolean): string => {
+    if (status) {
+        return '-mb-[1px] hover:text-primary-red px-6 font-semibold border-b-2 border-primary-red bg-red-100'
```

Comment:
```suggestion
        return 'border-b-2 border-primary-red bg-red-100'
```

Remove **-mb-[1px] hover:text-primary-red px-6 font-semibold** because you already returned it in line 29